python/trainers/ppo_models.py

```diff
@@ -307,7 +309,7 @@ def __init__(self, lr, brain, h_size, epsilon, max_step, normalize, use_recurren
                                             initializer=tf.zeros_initializer())
         self.sigma_sq = tf.exp(self.log_sigma_sq)
 
-        self.epsilon = tf.placeholder(shape=[None, a_size], dtype=tf.float32, name='epsilon')
+        self.epsilon = tf.random_normal(tf.shape(self.mu), dtype=tf.float32)
```

Comment:
Is there a way to manually change the standard deviation of this normal distribution ? I feel we should be able to tweak the level of noise here.